api-docs/apiLive.htm

```diff
@@ -33166,6 +33191,17 @@ <h2 class="flybar-button">Template</h2>
 								String which is searched on scoped resources
 							</td>
 						</tr>
+						<tr class=alt>
+							<td>
+								exactMatch
+								<br><i>(mandatory)</i>
```

Comment:
should not be mandatory, so that existing consumers of the api are not affected.

The field should be optional and default to false (so if you do not use it or pass it as a query parameter, exact match is automatically set to false)